### ISSUE TITLE
Package.getStylesheetsPath is deprecated.

### ISSUE THREAD

**Issue by andywenk** [johnotander/atom-underline-trailing-whitespace#2]
Store package style sheets in the `styles/` directory instead of `stylesheets/` in the `underline-trailing-whitespace` package

```
Package.getStylesheetsPath (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:460:9)
Package.getStylesheetPaths (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:471:32)
Package.loadStylesheets (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:453:38)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:185:19)
Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:165:15)
Package.load (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:179:12)
```

**Comment by andywenk** [comment]
I have changed the folder from stylesheets to styles in 

```
~/.atom/packages/underline-trailing-whitespace
```

The deprecation warning is gone after restarting Atom. Here is my config:

```
"*":
  editor:
    invisibles: {}
    showIndentGuide: true
    preferredLineLength: 120
    fontSize: 17
  core:
    themes: [
      "one-light-ui"
      "one-light-syntax"
    ]
    disabledPackages: [
      "minimap"
    ]
    projectHome: "/Users/andwen/project"
  "exception-reporting":
    userId: "****"
  welcome:
    showOnStartup: false
  metrics:
    userId: "****"
  minimap: {}
  rspec:
    command: "bundle exec rspec"
```

The package is still not working.

**Comment by MarioAraque** [comment]
+1

**Comment by johno** [comment]
Thanks for reporting this, it's now fixed in version 1.3.0.
